docs/overview/office-add-ins-community-call.md

```diff
@@ -11,7 +11,8 @@ The Microsoft Office Add-ins community call is a monthly call where you can lear
 
 ## Agenda for August 10th call
 
-The agenda for August will be posted about a week before the call.
+- OOXML libraries for JavaScript
+- Integrate your web site with Excel add-ins
 
 ## Previous calls
 
```

Comment:
@davidchesnut Curious if there's a link to the July call?

Reply:
I believe the July call post is currently with Linda for an edit pass.